api/src/main/resources/pdl/queries/hentPerson.graphql

```diff
@@ -96,5 +96,36 @@ query($ident: ID!){
                 opplysningsId
             }
         }
+        deltBosted{
```

Comment:
Er `matrikkelAdresse` noe vi burde hentet ut? Eller trenger vi ikke den her?

Reply:
Vet ikke om det er noe i `matrikkelAdresse` som vi trenger når vi henter ut fra `vegadresse`?

Reply:
i utgangspunktet skal alle matrikkeladresser (gårdsnr, bruksnummer........) ha vegadresse ref ein reform, men digihot kom over ein situasjon som lagde litt kluss: https://nav-it.slack.com/archives/C84H68ESC/p1616093199054300. Dette er ein ekstrem corner-case, men i god tradisjon så er det corner-casene som tar kontakt med nav.
Når eg sjekker kartverket så er det nokre prosent av matrikkeladressene som ikkje har vegadresse: https://www.kartverket.no/eiendom/adressering/status-vegadresser